src/main/webapp/components/workspace/workspace.js

```diff
@@ -236,6 +250,13 @@ export default () => {
                   })
                 }}
               />
+              <Filters
+                filters={filters}
+                setFilters={setFilters}
+                enable={enableFilters}
+                setEnable={setEnableFilters}
+                onApply={() => onSearchQuery(currentQuery)}
```

Comment:
Should we be rerunning the query on changes to the filter? In intrigue the filtering is done client side